### ISSUE TITLE
Attempting to docker compose up repair and getting: ModuleNotFoundError: No module named 'bencode3'

### ISSUE THREAD

**Issue by beanbaganalytica** [westsurname/scripts#12]
I am attempting to `docker compose --profile repair up` and I keep getting the below error. Dockerfile.scripts should be installing the module in the container, but it seems the debrid.py script can not find it.

```
repair  | Traceback (most recent call last):
repair  |   File "repair.py", line 4, in <module>
repair  |     import shared.debrid # Run validation
repair  |   File "/app/shared/debrid.py", line 5, in <module>
repair  |     import bencode3
repair  | ModuleNotFoundError: No module named 'bencode3'
```

**Comment by laster13** [comment]
reqirement.txt
bencode3==0.1.0 **#all**

**Comment by Pukabyte** [comment]
> reqirement.txt replace bencode3==0.1.0 #blackhole by bencode3==0.1.0 **#all**

I have made a pull request with these changes

**Comment by beanbaganalytica** [comment]
Closed in 1.3.1. Thanks Puka!
